ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/communication/ContinuousWalkingAPI.java

```diff
@@ -28,6 +28,7 @@ public class ContinuousWalkingAPI
    public static final ROS2Topic<ContinuousWalkingCommandMessage> CONTINUOUS_WALKING_COMMAND = IHMC_ROOT.withModule("continuous_walking").withType(ContinuousWalkingCommandMessage.class).withSuffix("command");
    public static final ROS2Topic<ContinuousWalkingStatusMessage> CONTINUOUS_WALKING_STATUS = IHMC_ROOT.withModule("continuous_walking").withType(ContinuousWalkingStatusMessage.class).withSuffix("status");
 
+   public static final ROS2Topic<PoseListMessage> PLACED_GOAL_FOOTSTEPS = IHMC_ROOT.withModule("continuous_walking").withType(PoseListMessage.class).withSuffix("placed_goal_footsteps");
```

Comment:
Why not include this in the CONTINUOUS_WALKING_COMMAND? There's joystick inputs in there too, this is another form of telling it where to go. Would be nice for the user reading the API to see all the options in one place.

Reply:
I didn't want to mess with the CONTINUOUS_WALKING_COMMAND right now. Isn't everything contained within  ContinuousWalkingAPI relavent to the API? What makes just the CONTINUOUS_WALKING_COMMAND the entire API?